percival/detector/adapter.py

```diff
@@ -31,9 +31,10 @@ def __init__(self, **kwargs):
 
         :param kwargs:
         """
+        self._log = logging.getLogger(".".join([__name__, self.__class__.__name__]))
         super(PercivalAdapter, self).__init__(**kwargs)
 
-        #logging.debug(kwargs)
+        #self._log.debug(kwargs)
```

Comment:
I guess you can either remove the comment or remove the line altogether?

Reply:
ok